### ISSUE TITLE
Accessibility: Video Element Missing Alt tag on user defined still image for lazy load

### ISSUE THREAD

**Issue by cornish-td** [soflyy/breakdance-bugs#516]
Getting pulled up on pagespeed insights and on WAVE a11y checker for having one alt tag not entered - this is for the video element. When set to lazy load and using my own thumbnail - despite the alt tag being set through the WP media library, it's not pulling it through. 

<img width="1009" alt="Screenshot 2022-11-07 at 10 01 52" src="https://user-images.githubusercontent.com/117569064/200282964-2cecc92d-5076-4c71-8f9d-8bd2df379a30.png">
<img width="311" alt="Screenshot 2022-11-07 at 10 04 44" src="https://user-images.githubusercontent.com/117569064/200282971-b69af79d-2d1a-451d-a01f-5f9e12524813.png">
<img width="289" alt="Screenshot 2022-11-07 at 10 04 37" src="https://user-images.githubusercontent.com/117569064/200282974-aab85415-5183-4c11-8afd-4615593bd9a2.png">

**Comment by cornish-td** [comment]
URL: https://bfrv2.onyx-sites.io/
Version 1.03 
WP 6.1

**Comment by a-dubiel** [comment]
Hey @cornish-td!

We recently made some improvements to the Video Element and changed the way we display video posters. We addressed this in the recent 1.1 Beta and looks like this element is passing accessibility requirements. In fact, we are currently working through all elements to make sure they are accessible. Feel free to download the 1.1 Beta or wait bit longer until we release 1.1 RC.

Thanks a lot for your report!

**Comment by a-dubiel** [comment]
✋🏻 Hey, thanks for your report. This has been addressed in the latest release.
